webui/react/src/components/GridListRadioGroup.tsx

```diff
@@ -0,0 +1,39 @@
+import { Radio } from 'antd';
+import { RadioChangeEvent } from 'antd/lib/radio';
+import React, { useCallback, useState } from 'react';
+
+import Icon from 'components/Icon';
+
+import css from './GridListRadioGroup.module.scss';
+
+export enum GridListView {
+  Grid,
+  List,
+}
+
+interface Props {
+  onChange?: (e: GridListView) => void;
+  value: GridListView;
+}
+
+const GridListRadioGroup: React.FC<Props> = ({ onChange, value }: Props) => {
+  const [ view, setView ] = useState(value);
+
+  const handleChange = useCallback((e: RadioChangeEvent) => {
+    if (onChange) onChange(e.target.value as GridListView);
+    setView(e.target.value as GridListView);
+  }, [ onChange ]);
+
+  return (
+    <Radio.Group className={css.base} value={view} onChange={handleChange}>
```

Comment:
nit visual thing, not sure if wanted/compromised?

<img width="212" alt="Schermata 2021-03-03 alle 09 48 03" src="https://user-images.githubusercontent.com/5413702/109778919-b0edfe80-7c05-11eb-8ff2-13769d969e8d.png">

Reply:
Great catch! I'll fix this.